cmd/portal/analytic/output.go

```diff
@@ -0,0 +1,68 @@
+package analytic
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/authgear/authgear-server/cmd/portal/util/google"
+	"google.golang.org/api/sheets/v4"
+)
+
+type OutputReportOptions struct {
+	OutputType                               string
+	GoogleOAuthClientCredentialsJSONFilePath string
+	GoogleOAuthTokenFilePath                 string
+	SpreadsheetID                            string
+	SpreadsheetRange                         string
+}
+
+func OutputReport(ctx context.Context, options *OutputReportOptions, data *ReportData) error {
+	switch options.OutputType {
+	case ReportOutputTypeStdout:
```

Comment:
I suggest we support CSV as the default output type. It is more useful than printing in Go-specific format.